### PR TITLE
Update bignumber.js dependency

### PR DESCRIPTION
**Pull request by axelchalon**
Updating bignumber.js to 4.1.0 on all repos, see https://github.com/parity-js/shared/pull/4#issuecomment-383031582

### CODE REVIEW

**Comment by amaury1093** [comment]
Can you do a `npm install @parity/abi` so that it updates the ref to abi in package-lock.json too?

**Comment by axelchalon** [comment]
Yes! Done

File: package.json [hunk 1]
```diff
@@ -69,10 +69,10 @@
     "sinon-chai": "2.8.0"
   },
   "dependencies": {
-    "@parity/abi": "2.1.x",
+    "@parity/abi": "^2.1.4",
```
Comment (amaury1093) [inline package.json]: Really small detail (again). Change this to 2.1.x or ~2.1.4 (^2.1.4 signifies something else)

Comment (axelchalon) [reply package.json]: Sure!

**Review by amaury1093 — APPROVED** [review]

**Pull request merged**